saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/EntityRelation/EntityRelationSensors.scala

```diff
@@ -20,16 +20,16 @@ object EntityRelationSensors {
   lazy val personGazetSensor = new GazeteerReader("gazeteer/known_maleFirst.lst", "Gaz:Person", true, true)
   personGazetSensor.addFile("gazeteer/known_femaleFirst.lst", true, true)
 
-  def readConllData(dir: String): (List[ConllRawSentence], List[ConllRelation], List[ConllRawToken]) = {
-    val reader = new Conll04_Reader(dir, "Token")
+  def readConllData(dir: String, sentenceIdOffset: Int = 0): (List[ConllRawSentence], List[ConllRelation], List[ConllRawToken]) = {
+    val reader = new Conll04_Reader(dir, "Token", sentenceIdOffset)
```

Comment:
I can see only a very limited history of this, but if I have changed this and have added this offset then the answer is YES. Otherwise I need to go back and check the `populate` thing.